physionet-django/physionet/settings/base.py

```diff
@@ -30,6 +30,7 @@
 ENABLE_SSO = config('ENABLE_SSO', default=False, cast=bool)
 SSO_REMOTE_USER_HEADER = config('SSO_REMOTE_USER_HEADER', default='HTTP_REMOTE_USER')
 SSO_LOGIN_BUTTON_TEXT = config('SSO_LOGIN_BUTTON_TEXT', default='Login')
+ENABLE_SESSION_SECURITY = config('ENABLE_SESSION_SECURITY', default=True, cast=bool)
```

Comment:
```suggestion
ENABLE_SESSION_SECURITY = config('ENABLE_SESSION_SECURITY', default=False, cast=bool)
```
I'd keep this `False` to not induce any changes into the MIT deployment. We should treat their config as default.